app/src/test/scala/org/alephium/explorer/persistence/DatabaseFixture.scala

```diff
@@ -25,7 +25,7 @@ import slick.jdbc.JdbcProfile
 
 import org.alephium.util.Duration
 
-trait DatabaseFixture {
+trait DatabaseFixture extends AutoCloseable {
```

Comment:
Ah my bad. I should've documented this in the PR. Initially I had the test-cases using the `AutoCloseable` similar to Benchmark [test-cases](https://github.com/alephium/explorer-backend/blob/master/benchmark/src/test/scala/org/alephium/explorer/benchmark/db/state/WriteBenchmarkStateSpec.scala#L62) so the connection gets closed after each run (#146). 

```scala
import org.alephium.explorer.util.TestUtils.using

it should "blah!" in {
  using(new Fixture) { fixture =>
    import fixture._

    import config.profile.api._

    runAction(blahTable.result)
  }
}
```
The `run` function in `DBRunner` had to be renamed to `runAction` because it conflicted with ScalaTest's `run` function. The renaming was changing a lot of files and made this PR much larger. So I removed using, `using` but left the change (first two commits).

Should I remove `AutoCloseable` and move `TestUtils` back to Benchmark module?

Reply:
maybe you can remove the `AutoCloseable` as it's not used yet, but let's keep the `TestUtils`, no need to move it back.
I'll have a look but maybe we could add an `afterEach` or something like this in our `DatabaseFixture` to do the close.